nomad/search_endpoint.go

```diff
@@ -0,0 +1,151 @@
+package nomad
+
+import (
+	"fmt"
+
+	memdb "github.com/hashicorp/go-memdb"
+	"github.com/hashicorp/nomad/nomad/state"
+	st "github.com/hashicorp/nomad/nomad/structs"
```

Comment:
Can you keep it as structs. Avoid named imports unless there is a conflict or the name is terribly long